ExtraDry/ExtraDry.Blazor/Models/PropertyDescription.cs

```diff
@@ -43,6 +43,7 @@ public PropertyDescription(PropertyInfo property)
         }
         --recursionDepth;
         PropertyType = Property.PropertyType.IsGenericType && Property.PropertyType.GetGenericTypeDefinition() == typeof(Nullable<>) ? Property.PropertyType.GetGenericArguments()[0] : Property.PropertyType;
+        InputType = Property.GetCustomAttribute<InputFormatAttribute>()?.DataTypeOverride ?? PropertyType;
```

Comment:
Mismatch between names "InputType" and "InputFormat", can it just be called InputFormat?

Reply:
It can be. This is the InputFormats override type or the actual type of the entity, so it's the Type of the control to render. Happy to align the names, my thought on not aligning is that the InputFormat Attribute may eventually have multiple properties to change the input and didn't want confusion

Reply:
I've renamed to match